File: task7/src/test/java/framework/utils/Reader.java
package framework.utils;

import aquality.selenium.logger.Logger;

import java.io.FileInputStream;
import java.io.IOException;
import java.util.Properties;

public class Reader {

    private static final Logger logger = Logger.getInstance();

    public static String getParameter(String parameter) throws IOException {
        logger.info("Get parameter " + parameter);
        Properties property = new Properties();
        try {
            FileInputStream fileInputStream = new FileInputStream("src/test/resources/cloudinary.properties");
            property.load(fileInputStream);
        } catch (IOException e) {
            throw new IOException("Parameter not found");
        }
        return property.getProperty(parameter);
    }
}

File: KotlinSecretCloset/src/main/kotlin/framework/utils/Reader.kt
package framework.utils

import aquality.selenium.logger.Logger
import java.io.FileInputStream
import java.util.*

private const val pathToConfig: String = "src/test/resources/config.properties"
val logger: Logger = Logger.getInstance()

fun getParameter(parameter: String?): String? {
    logger.info("Get $parameter  from config file")
    val property = Properties()
    property.load(FileInputStream(pathToConfig))
    return property.getProperty(parameter)
}
